### ISSUE TITLE
Some question about pretrained model.

### ISSUE THREAD

**Issue by HardWayShip** [bismex/MetaBIN#9]
Hello!

I found that there are two pre-training models in Section 6, and **I want to ask which model should I import if we want to achieve the best results**?

looking forward to your answer, thanks!    :)

**Comment by bismex** [comment]
Hello! :D

The w1.4 model shows better performance than the w1.0 model.
If you want to check the performance according to the model, please check Table 6 (page 14) of the paper! (https://arxiv.org/pdf/2011.14670.pdf, Supplementary Material)

The performance below is a summary of Table 6, and this shows the average rank-1 accuracy evaluated on four datasets (VIPeR, PRID, GRID, i-LIDS).
- MobileNetV2 (w1.0) 61.9%
- MobileNetV2 (w1.4) 64.7% 
- ResNet18 59.5%
- ResNet34 62.8%
- ResNet50 66.0%
- ResNet101 68.1%
